Tests/AK/TestOptional.cpp

```diff
@@ -122,15 +122,18 @@ TEST_CASE(comparison_with_numeric_types)
 
 TEST_CASE(test_copy_ctor_and_dtor_called)
 {
-#ifdef AK_HAVE_CONDITIONALLY_TRIVIAL
     static_assert(IsTriviallyDestructible<Optional<u8>>);
-    static_assert(IsTriviallyCopyable<Optional<u8>>);
+    // FIXME: Figure out what is missing for this one:
+    // static_assert(IsTriviallyCopyable<Optional<u8>>);
```

Comment:
Is this still an issue?

Reply:
That should be removed in the other Optional commit, I can remove the comment early though